packages/core/src/diff/changes/argument.ts

```diff
@@ -39,12 +40,19 @@ export function fieldArgumentDefaultChanged(
     type: ChangeType.FieldArgumentDefaultChanged,
     message:
       typeof oldArg.defaultValue === 'undefined'
-        ? `Default value '${newArg.defaultValue}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
-        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${oldArg.defaultValue}' to '${newArg.defaultValue}'`,
+        ? `Default value '${safeString(newArg.defaultValue)}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
+        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${safeString(oldArg.defaultValue)}' to '${safeString(newArg.defaultValue)}'`,
     path: [type.name, field.name, oldArg.name].join('.'),
   };
 }
 
+function safeString(obj: any) {
```

Comment:
suggestion: Let's move this to [core/src/utils/string.ts](https://github.com/kamilkisiela/graphql-inspector/blob/0575549b0a2dfc86ed523e01b718d30f3e426cea/packages/core/src/utils/string.ts) and import it here